### DISCUSSION TITLE
【BUG】关于无序清单标签的使用问题

### DISCUSSION THREAD

**Discussion by ottopan** [Quorafind/Obsidian-Thino#602] [General 通用]
两个问题
1. 每行都有单独敲一遍[- ]符号才能触发无序清单，没有继承的逻辑
2. [tab]键无法触发无序清单的缩进， 只能触发聚焦 其他页面元素

**Comment by saralaaga** [comment]
我也遇到了这个问题，似乎是mac系统的一个问题？我的是m2芯片的mac，之前还好好的，但是不知道是插件冲突还是怎样，我现在在使用无序列表的时候也必须敲一下 [-] 令我非常迷惑。
这类小bug虽然不影响使用，但让整体体验糟糕。

**Comment by juestchaos** [comment]
这是一个插件冲突，但是由于目前只看到 conversation 中提到，且未提供插件冲突信息。

建议：单独使用 thino 并重启，看看是否还存在问题。如果不存在，则是插件间冲突导致的
